lib/stat_tracker.rb

```diff
@@ -34,6 +34,38 @@ def lowest_total_score
   end
 
 #------------LeagueStatistics
+
+
+#------------SeasonStatistics
+
+  def coach_list_wins_losses
+    @coach_hash = Hash.new
+    game_teams.each do |gt|
+      (@coach_hash[gt.head_coach] ||= []) << gt.result
+    end
+    @coach_hash
+  end
```

Comment:
Awesome job here! Two quick changes I would make

1. You can remove the @ symbol and just use normal variables
2. Since this is a helper method you can move it into the private section